src/auth-service/routes/v1/users.js

```diff
@@ -327,26 +328,42 @@ router.post(
         .withMessage("this is not a valid email address")
         .trim(),
       body("organization")
-        .exists()
-        .withMessage("organization is missing in your request")
+        .optional()
+        .notEmpty()
+        .withMessage("organization should not be empty if provided")
         .bail()
         .trim(),
       body("long_organization")
-        .exists()
-        .withMessage("long_organization is missing in your request")
+        .optional()
+        .notEmpty()
+        .withMessage("long_organization should not be empty if provided")
         .bail()
         .trim(),
       body("privilege")
-        .exists()
-        .withMessage("privilege is missing in your request")
+        .optional()
+        .notEmpty()
+        .withMessage("privilege should not be empty if provided")
         .bail()
         .isIn(["admin", "netmanager", "user", "super"])
         .withMessage("the privilege value is not among the expected ones")
         .trim(),
+      body("password")
+        .exists()
```

Comment:
@FaithDaka , the password is a now a mandatory requirement at the input validation level.